app.devsuite.Ptyxis.json

```diff
@@ -0,0 +1,98 @@
+{
+    "app-id" : "app.devsuite.Ptyxis",
```

Comment:
If you can mention this project and author somewhere in https://devsuite.app/ that'd be nice.

Reply:
Done

Reply:
I don't see anything?

![image](https://github.com/flathub/flathub/assets/62639087/bf980a38-0a52-42dc-9cb4-d2f2c3a075ba)

Reply:
Ah I can see it now. Thanks